packages/fast-usdc/src/type-guards.js

```diff
@@ -166,16 +165,6 @@ export const FeedPolicyShape = M.splitRecord(
 );
 harden(FeedPolicyShape);
 
-/** @type {TypedPattern<FastUSDCConfig>} */
-export const FastUSDCConfigShape = M.splitRecord({
-  terms: FastUSDCTermsShape,
-  oracles: M.recordOf(M.string(), M.string()),
-  feeConfig: FeeConfigShape,
-  feedPolicy: FeedPolicyShape,
-  chainInfo: M.recordOf(M.string(), CosmosChainInfoShape),
```

Comment:
what was the symptom that this commit is fixing?

Reply:
```
 message: `Command failed: /home/connolly/projects/agoric-sdk/packages/agoric-cli/src/entrypoint.js run /home/connolly/projects/agoric-sdk/packages/fast-usdc-deploy/src/start-fast-usdc.build.js --net MAINNET␊
    agoric: run: running /home/connolly/projects/agoric-sdk/packages/fast-usdc-deploy/src/start-fast-usdc.build.js␊
    agoric: (Error#1)␊
    Error#1: chainInfo: ethereum: [1]: {"chainId":"eip155:1","namespace":"eip155","reference":"1"} - Must have missing properties ["bech32Prefix"]␊

Reply:
It's a drive-by - leftover from #11170 and/or #11131. 

The symptoms - we're unable to use `start-fast-usdc.build.js`. This proposal won't be used on mainnet again but some folks use it for testing:

```
 message: `Command failed: /home/connolly/projects/agoric-sdk/packages/agoric-cli/src/entrypoint.js run /home/connolly/projects/agoric-sdk/packages/fast-usdc-deploy/src/start-fast-usdc.build.js --net MAINNET
    agoric: run: running /home/connolly/projects/agoric-sdk/packages/fast-usdc-deploy/src/start-fast-usdc.build.js
    agoric: (Error#1)
    Error#1: chainInfo: ethereum: [1]: {"chainId":"eip155:1","namespace":"eip155","reference":"1"} - Must have missing properties ["bech32Prefix"]
```

Reply:
Thanks. Not necessary now but consider including that in the commit message. (I should do that more too when making "fix" commits)